platform/qt/src/qmapboxgl.cpp

```diff
@@ -562,21 +621,21 @@ Coordinate QMapboxGL::coordinate() const
     return Coordinate(latLng.latitude, latLng.longitude);
 }
 
-void QMapboxGL::setCoordinate(const Coordinate &coordinate_)
+void QMapboxGL::setCoordinate(const QMapbox::Coordinate &coordinate_)
 {
     d_ptr->mapObj->setLatLng(mbgl::LatLng { coordinate_.first, coordinate_.second }, d_ptr->margins);
 }
 
 /*!
-    Convenience method for setting the \a coordinate and \a zoom simultaneously.
+    Convenience method for setting the \a coordinate_ and \a zoom_ simultaneously.
```

Comment:
Ah, that’s unfortunate. In Objective-C and Swift, it’s also possible for a getter and a variable to share the same namespace, and `_` always denotes an internal implementation detail that shouldn’t be touched. So one old convention is to say e.g. `theCoordinate` or `aZoom`, but I don’t know if that looks reasonable in Qt code.

Is it possible to document this method on its declaration rather than its implementation? Then you can give the parameter two different names, `coordinate` in the header (which shouldn’t conflict with `coordinate()`) and `coordinate_` here in the implementation.